proto/keystore_api/v1/keystore.proto

```diff
@@ -116,12 +116,32 @@ message CreateInviteRequest {
     uint64 created_ns = 3;
 }
 
+// Request to create an invite payload from an existing topic, and store the topic keys in the Keystore
+message CreateInviteFromTopicRequest {
+    xmtp.message_contents.InvitationV1.Context context = 1;
```

Comment:
Do we actually need the context here? The Keystore already should already have the `context` in its records, if it knows the `content_topic`. Same for `created_ns`.

Reply:
Ya, I think you're right. I can remove these.